### PR TITLE
History filter - #444

### PR DESCRIPTION
**Pull request by gokulprathin8**
PR for https://github.com/ESIPFed/Geoweaver/issues/444

### CODE REVIEW

File: src/main/resources/static/js/gw.history.js [hunk 1]
```diff
@@ -205,7 +205,39 @@ GW.history = {
      */
     getProcessHistoryTable: function(msg){
 
-        var content = `<table class=\"table table-color\" id=\"process_history_table\"> 
+        let content = `
+        <div class="history-filters" style="display: flex; margin: 10px; justify-content: space-between;">
+            <div id="recentDaysFilterContainer">
+                <label for="recentDaysFilter">Recent Days:</label>
```
Comment (ZihengSun) [inline src/main/resources/static/js/gw.history.js:211]: The other two filters make sense. But this one is a little confusing. The current records are aligned based on the time. People can see the recent records directly. No need to filter using last n days here. Can you remove?

**Review by ZihengSun — CHANGES_REQUESTED** [review]
One comment

File: src/main/resources/static/js/gw.history.js (hunk 1 above)
Comment (gokulprathin8) [reply src/main/resources/static/js/gw.history.js:211]: ok sure

**Comment by gokulprathin8** [comment]
done

**Review by ZihengSun — APPROVED** [review]

**Pull request merged**